galaxy/importer/loaders/apb.py

```diff
@@ -39,15 +63,27 @@ def load(self):
         with open(os.path.join(self.path, self.metadata_file)) as fp:
             metadata = yaml.safe_load(fp)
 
+        meta_parser = APBMetaParser(metadata, logger=self.log)
+
         try:
             name = metadata['name']
         except KeyError:
             raise exc.ContentLoadError('Missing "name" field in metadata')
+
+        try:
+            description = metadata['description']
+        except KeyError:
+            raise exc.ContentLoadError('Missing "description" field in metadata')
```

Comment:
> Also should probably specify the error was while importing APB metadata.

> I would also be tempted to add ContentLoaderError subclasses for each of the content types 
While I'm for adding context to importer exceptions, I don't think creating a subclass for each content type can be any useful. However it might be reasonable to create for example subclass like `MetadataParsingError`.

Reply:
Subclassed ContentLoaderError as suggested. Not touch other loaders, as they are outside the scope of this PR.

Not adding the metadata file name, as it is surfaced in the output log at the start of the load process, so in theory it should be apparent to the user which file is in question when an exception is thrown.